babel-jest-root-mode-wrapper.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('babel-jest').createTransformer({
```

Comment:
Also see:
https://github.com/FormidableLabs/enzyme-matchers/pull/315/files#diff-27356c588e80256290a0c5b661a4e518R42
https://github.com/FormidableLabs/enzyme-matchers/pull/315/files#diff-23285899aef99471ba8fbfac42c9f9b6R53
https://github.com/FormidableLabs/enzyme-matchers/pull/315/files#diff-7ef917cb23a113e5a65f8047120b86dbR63

I am not sure that this is the right solution. It's taken from here https://babeljs.io/docs/en/config-files#jest.